src/App.js

```diff
@@ -4,7 +4,7 @@ import './index.css'
 // import * as serviceWorker from './serviceWorker'
 
 import Dashboard from './Components/Dashboard/Dashboard'
-import Overview from './Pages/Overview/Overview'
+import Overview from './Pages/Overview/App'
```

Comment:
change the filename `App.js` back to `Overview.js` or something more descriptive